### PR TITLE
Enable running tests with `gradle test`

### PR DESCRIPTION
**Pull request by rohansingh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,9 +142,7 @@ s3mper.metastore.name|ConsistentListingMetastore|The name of the DynamoDB table
 Verification
 ------------
 
-A unit test is included with the library that exercises the advised commands on the file system.  Build the test jars with the command `./gradlew testJar` and include the `s3mpter-test.jar` in the classpath.  Use the `scripts/verify-consistent-listing.sh` command to run the unit tests.  
-
-__Note:__ you may need to modify some paths in the script to point to the correct directories.
+Run `gradle test`. You will need to set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
```
Comment (steffengr) [inline README.md:145]: Shouldn't this run with BigTable now?

File: test/java/com/netflix/bdp/s3mper/metastore/impl/InMemoryMetastoreTest.java [hunk 2]
```diff
@@ -41,16 +37,22 @@
...
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
 
 /**
  * @author liljencrantz@spotify.com
  */
 public class InMemoryMetastoreTest {
+
+    private static final String AWS_ACCESS_KEY_ID = "AWS_ACCESS_KEY_ID";
```
Comment (steffengr) [inline test/java/com/netflix/bdp/s3mper/metastore/impl/InMemoryMetastoreTest.java:52]: Why does the in-memory one need AWS credentials? It should not need a database.

**Comment by steffengr** [comment]
👍

File: test/java/com/netflix/bdp/s3mper/metastore/impl/InMemoryMetastoreTest.java (hunk 2 above)
Comment (rohansingh) [inline test/java/com/netflix/bdp/s3mper/metastore/impl/InMemoryMetastoreTest.java:52]: AWS credentials are used to write to the S3 bucket in this test.

**Pull request merged**